src/executor/mod.rs

```diff
@@ -1,5 +1,7 @@
 //! Types which can take a boxed closure and execute it, preferably in the background.
 
+#![cfg_attr(feature="clippy", allow(boxed_local))]
```

Comment:
It looks similar to this no? https://github.com/Manishearth/rust-clippy/issues/916